src/components/generics/vacation-test-forms/employee-vacation-requests.tsx

```diff
@@ -92,6 +105,68 @@ const RenderEmployeeVacationRequests = ({ persons }: Props) => {
     initializeRequests();
   }, [person]);
 
+  /**
+   * Initializes all vacation request statuses
+   */
+  const initializeRequestStatuses = async () => {
+    try {
+      const vacationRequestStatuses: VacationRequestStatus[] = [];
+      const statusesApi = Api.getVacationRequestStatusApi(accessToken?.access_token);
+
+      await Promise.all(requests.map(async request => {
+        const createdStatuses = await statusesApi.listVacationRequestStatuses({ id: request.id! });
+        createdStatuses.forEach(createdStatus => {
+          vacationRequestStatuses.push(createdStatus);
+        });
+      }));
+
+      setStatuses(vacationRequestStatuses);
+      setLoading(false);
+    } catch (error) {
+      context.setError(strings.errorHandling.fetchVacationDataFailed, error);
+    }
+  };
+
+  useEffect(() => {
+    if (requests.length <= 0) {
```

Comment:
Is there any reason to expect a length of less than 0, if not then requests.length === 0 should be fine as a check. Even better would be the check (!array.length) although you should double check this works as intended.

Reply:
I think both can be safely included, still works as intended.

```suggestion
      if (requests.length === 0 || !requests.length) {
      return;
    }
```